### PR TITLE
Create vets, spec and visits tables

### PR DESCRIPTION
**Pull request by blueberry1312**
##  In this PR, I did the following changes:

- Created new vets table.
- Created two 'joins' table (specializations and visits) to handle relationship.s between others tables.
- Added data to the tables created  (vets, specializations and visits)
- Add queries to answer the following questions:
- Who was the last animal seen by William Tatcher?
image
- How many different animals did Stephanie Mendez see?
image
- List all vets and their specialties, including vets with no specialties.
image
- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
image
- What animal has the most visits to vets?
image
- Who was Maisy Smith's first visit?
image
- Details for most recent visit: animal information, vet information, and date of visit.
image
- How many visits were with a vet that did not specialize in that animal's species?
image
- What specialty should Maisy Smith consider getting? Look for the species she gets the most.
Digimon:
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -106,3 +106,83 @@ SELECT COUNT(*) as Number_of_animals, o.full_name as owner
...
+  SELECT *
+  FROM specializations spec
+  WHERE spec.vet_id = vet.id AND spec.specie_id = animal.species_id
+);
+
+-- Nine
+SELECT COUNT(s.name) as Count, s.name as Specie
+FROM visits vis 
+JOIN vets vet ON vis.vet_id = vet.id
+JOIN animals an ON vis.animal_id = animal.id
```
Comment (Davidosky007) [inline queries.sql]: * I get an error when I run this query, kindly fix it 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @blueberry1312 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (blueberry1312) [reply queries.sql]: Oh, i dont know why happend that, but i fix now, I only need to change 'an' by 'animal', thanks for the feedback @Davidosky007

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED 👍 

Hi @blueberry1312 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Query error fixed
:heavy_check_mark:  Good commit messages

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**